_data/roles.yml

```diff
@@ -0,0 +1,89 @@
+- name: Registry Metadata Steward
+  open: true
+  commitment: 2-4 hours per month
+  requirements:
+    - Love for metadata
+    - basic python
+    - community mindset
+  description: |
+    You are responsible for shepherding our registry
+    metadata according to our principles and SOPs, merge changes and generally
+    protect and guard them. This involves also facilitating the implementation of QC
+    checks.
+- name: OBO NOR Dashboard maintainer
+  open: true
+  commitment: 2-6 hours per month
+  requirements:
+    - Basic knowledge of dashboard
+    - Social skills are a nice to have
+    - Infinite patience also helpful
+  description: |
+    You will be responsible for registering new ontologies at the OBO dashboard and helping their
+    owners pass them.
+- name: OBO Website Coordinator
+  open: true
+  commitment: 2-6 hours per month
+  requirements:
+    - Working knowledge of javascript
+    - Working knowledge of python
+    - Nice to have: knowledge of jekyll and liquid (like jinja)
+    - Should have technical background
+  description: |
+    You will be responsible for gatekeeping community changes to the website.
+- name: OBO Dashboard Maintainer
+  open: true
+  commitment: 2-10 hours per month
+  requirements:
+    - Basic knowledge of dashboard
+    - Social skills are a nice to have
+    - Infinite patience also helpful
+  description: |
+    Updating the [OBO Dashboard](http://dashboard.obofoundry.org/dashboard/index.html) once per month. 
+    Fixing horrible bugs in OBO-Dashboard if they come up (should be rare). 
+    Working with OBO Ops to find people for extending dashboard when new principles come along.
+- name: Technical Working Group OBO Operations liaison
+  open: true
+  commitment: 2 hours per month
+  requirements:
+    - Strong python skills
+    - Some level of pro-activity helpful
+    - Experience with ODK/docker
+  description: |
+    Preparing a short report about all general developments in OBO Technical Working Group (TWG)
+     for the OBO Operations team meeting. You will go through pull requests and emails and summarize 
+     in bullet points and report to OBO Ops.
+- name: PURL system maintainer
+  open: false
+  description: |
+    PURL config curation and PURL server maintenance.
+  lead(s): James Overton
+- name: OMO metadata coordinator
+  open: false
+  description: |
+    You will be responsible for registering new ontologies at the OBO dashboard and helping their
+    owners pass them.
+  lead(s): 
+    - Jie Zheng and 
+    - Nico Matentzoglu
```

Comment:
next step is to turn these lists of strings into lists of dictionaries so we can include extra information about each person

```yaml
lead(s):
 - name: Jie Zheng
   status: [lead / support]
 - name: Nico Matentzoglu
   status: [lead / support]
```

Reply:
cool. I added this where I knew the info.

@jamesaoverton @zhengj2007 @nlharris @matentzn could you please add the data for yourselves, where appropriate? thanks

Reply:
I added my status and dates